src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2654,6 +2657,102 @@ private Process StartWithShellExecute(ProcessStartInfo startInfo)
         #endregion
     }
 
+    /// <summary>
+    /// Provides argument completion for Verb parameter.
+    /// </summary>
+    public class VerbArgumentCompleter : IArgumentCompleter
+    {
+        private const string StartProcessCommandName = "Start-Process";
+        private const string FilePathParameterName = "FilePath";
+        private const string GetCommandCommandName = "Get-Command";
+        private const string CommandNameParameterName = "Name";
+        private const string CommandTypeParameterName = "CommandType";
```

Comment:
If these consts are used once this reduces readability,  I'd remove them.

Reply:
Removed.